src/env.cc

```diff
@@ -1092,6 +1092,13 @@ void Environment::InitializeLibuv() {
 void Environment::ExitEnv(StopFlags::Flags flags) {
   // Should not access non-thread-safe methods here.
   set_stopping(true);
+
+#if HAVE_INSPECTOR
+  if (inspector_agent_ && inspector_agent_->IsWaitingForConnect()) {
```

Comment:
`StopWaitingForConnect` checks `IsWaitingForConnect` anyways. IMHO, there is no need for `IsWaitingForConnect` public method at all, just unconditionally call `Stop`. Might make sense to rename to something like `StopIfWaitingForConnect`

Reply:
Good catching, thanks. Updated.